### PR TITLE
feat(typescript): add definition file for TS users

### PR DESCRIPTION
**Pull request by michael-zapata**
http://ci.ironmann.io/gh/scality/Integration/2367#build-parameters End to end passed without any problem.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1 @@
-module.exports = require('./lib/Logger.js');
+module.exports = { Logger: require('./lib/Logger.js') };
```
Comment (AntoninCoulibaly) [inline index.js]: Why did you change that, is it not simpler to call the logger the same way as before ?

**Comment by AntoninCoulibaly** [comment]
Just a little comment, overall :+1:

File: index.js (hunk 1 above)
Comment (michael-zapata) [inline index.js]: cf #75, we didn't export an object, nor a default field, which made the ES6 import fizzle.

**Comment by rahulreddy** [comment]
I guess this is the price we pay for using ES5 exports with ES6 imports. Do you mind separating this change into a new PR?

**Comment by michael-zapata** [comment]
Merging on behalf of @rahulreddy to get things going forward.
